examples/01_workflows_example.ipynb

```diff
@@ -34,7 +34,19 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
This notebook needs to be re-run

Reply:
Reran in commit [d30939e](https://github.com/entropicalabs/openqaoa/pull/179/commits/d30939e8b1dcf1d010ddd3a0dccb58a69de4e758)